File: Apps4Bro.Windows/AdWrapper.cs
namespace Apps4Bro
{
    /// <summary>
    /// Ad instance. Also contains stat data
    /// </summary>
    public class AdWrapper
    {
        private readonly AdNetworkHandler m_adNetworkHandler;
        private readonly string m_id;
        private readonly string m_name;

        public int SuccessCount { get; set; }
        public int FailCount { get; set; }
        public int CallCount { get; set; }
        public int ClickCount { get; set; }
        public int ShowCount { get; set; }
        public string LastError { get; set; }

        public AdNetworkHandler AdNetworkHandler
        {
            get { return m_adNetworkHandler; }
        }

        public string Id
        {
            get { return m_id; }
        }

        public string Name
        {
            get { return m_name; }
        }

        public AdWrapper(AdNetworkHandler handler, string id, string name)
        {
            m_adNetworkHandler = handler;
            m_id = id;
            m_name = name;
        }
    }
}

File: Apps4Bro.Android/old/src/net/runserver/apps4bro/AdWrapper.java
package net.runserver.apps4bro;

public class AdWrapper
{
    private final AdNetworkHandler m_adNetworkHandler;
    private final String m_id;
    private final String m_name;

    public int SuccessCount;
    public int FailCount;
    public int CallCount;
    public int ClickCount;
    public String LastError;
    public AdNetworkHandler.AdObject AdRequest;

    public AdNetworkHandler getNetworkHandler()
    {
        return m_adNetworkHandler;
    }

    public String getId()
    {
        return m_id;
    }

    public String getName()
    {
        return m_name;
    }

    public AdWrapper(AdNetworkHandler handler, String id, String name)
    {
        m_adNetworkHandler = handler;
        m_id = id;
        m_name = name;
    }
}
